packages/medusa/src/api-v2/store/carts/[id]/line-items/[line_id]/route.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  deleteLineItemsWorkflow,
+  updateLineItemInCartWorkflow,
+} from "@medusajs/core-flows"
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { ICartModuleService } from "@medusajs/types"
+import { MedusaError, remoteQueryObjectFromString } from "@medusajs/utils"
+import { MedusaRequest, MedusaResponse } from "../../../../../../types/routing"
+import { defaultStoreCartFields } from "../../../query-config"
+import { StorePostCartsCartLineItemsItemReq } from "./validators"
+
+export const POST = async (req: MedusaRequest, res: MedusaResponse) => {
+  const cartModuleService = req.scope.resolve<ICartModuleService>(
+    ModuleRegistrationName.CART
+  )
+
+  const cart = await cartModuleService.retrieve(req.params.id, {
+    select: ["id", "region_id", "currency_code"],
+    relations: ["region", "items", "items.variant_id"],
+  })
```

Comment:
suggestion: should we use the remote query here?

Reply:
I chose to use the module service directly, as I don't need any other resources than what it internal to the cart module. Any specific reason you want to use the remote query?